content/docs/command-reference/commit.md

```diff
@@ -17,65 +16,56 @@ positional arguments:
-  of the files/directories used by the stage as dependencies or created as
-  outputs are missing from `dvc.yaml`. It is possible to
-  [add missing data to an existing stage](/docs/user-guide/how-to/add-deps-or-outs-to-a-stage),
-  and then `dvc commit` can be used to save outputs to the cache (and update
-  `dvc.lock`)
-
-- It's always possible to manually execute the command or source code used in a
-  stage without DVC (outputs must be unprotected or removed first in certain
-  cases, see `dvc unprotect`). Once the desired result is reached, use
-  `dvc commit` to update the `dvc.lock` file(s) and store changed data to the
-  cache.
-
-Let's take a look at what is happening in the first scenario closely. Normally
-DVC commands like `dvc add`, `dvc repro` or `dvc run` commit the data to the
-<abbr>cache</abbr> after creating or updating a `dvc.lock` or `.dvc` file. What
-_commit_ means is that DVC:
-
-- Computes a hash for the file/directory.
-- Enters the hash value and file name in the `dvc.lock` or `.dvc` file.
-- Tells Git to ignore the file/directory (adding them to `.gitignore`). (Note
-  that if the <abbr>project</abbr> was initialized with no Git support
-  (`dvc init --no-scm`), this does not happen.)
-- Adds the file(s) in question to the cache.
-
-There are many cases where the last step is not desirable (for example rapid
-iterations on an experiment). The `--no-commit` option prevents it (on the
-commands where it's available). The file hash is still computed and added to the
-`dvc.lock` or `.dvc` file, but the actual data is not cached. And this is where
-the `dvc commit` command comes into play: It performs that last step when
-needed.
-
-Note that it's best to avoid the last three scenarios. They essentially
-force-update the `dvc.lock` or `.dvc` files and save data to cache. They are
-still useful, but keep in mind that DVC can't guarantee reproducibility in those
-cases.
+Stores the current contents of files and directories tracked by DVC in the
+<abbr>cache</abbr>, and updates `dvc.lock` or `.dvc` files if/as needed. This
+forces DVC to accept any changed contents of tracked data currently in the
+<abbr>workspace</abbr>. We explore the scenarios in which this can be useful
+next.
+
+💡 For convenience, a pre-commit Git hook is available to remind you to
+`dvc commit` when needed. See `dvc install` for more info.
+
+Mainly, `dvc commit` provides a way to complete DVC commands that track data
+(`dvc add`, `dvc repro`, `dvc import`, etc.), when they have been used with the
+`--no-commit` or `--no-exec` options. Those options cause the commands to skip
+these step(s) of the tracking process:
+
+- Save the hash value of the file(s) in the `dvc.lock` or `.dvc` file.
+- Store the file(s) contents in the cache.
+
+> Skipping these steps is typically done to avoid caching unfinished data, for
+> example when exploring different data or
+> [stages](/doc/command-reference/run)).
+
+Other scenarios for `dvc commit` include:
```

Comment:
it still reads that they are different from the Note above ... which is not exactly true..

also, do we need it to be a Note? why is it a Note?

Reply:
> still reads that they are different from the Note above

I don't see this so I'm not sure how to fix that but happy to make any specific changes you have in mind.

> do we need it to be a Note? why is it a Note?

Maybe we don't need it (no strong opinion, please lmk) but it explains why people would skip steps during add/repro/import, etc., since that's what we're explaining as the main use for `commit`. It's a note because it's about those other commands, not about `commit`.

Reply:
> but happy to make any specific changes you have in mind.

Other -> Specifically, scenarios for ... 

It looks like that is the meaning (at least from the content)

Reply:
Changed!